tests/pytest/oauth/test_views.py

```diff
@@ -0,0 +1,56 @@
+from django.http import HttpResponse
+from django.urls import reverse
+
+from benefits.core import session
+from benefits.oauth.views import logout, post_logout, _generate_redirect_uri
+
+import pytest
+
+
+@pytest.mark.request_path("/oauth/logout")
+def test_logout(mocker, session_request):
+    # logout internally calls _deauthorize_redirect
+    # this mocks that function and a success response
+    # and returns a spy object we can use to validate calls
+    message = "logout successful"
+    spy = mocker.patch("benefits.oauth.views._deauthorize_redirect", return_value=HttpResponse(message))
```

Comment:
This works, but could we mock `load_server_metadata()` instead and then check the `Location` header on the `response` for a more comprehensive test?

Reply:
That feels like a different test, of the `_deauthorize_redirect` function. This test is ensuring our logout function is handling app sign out correctly.

I will write another test for that function 👍

Reply:
Added in [`3f8db89`](https://github.com/cal-itp/benefits/pull/436/commits/3f8db89cfd997dae4c897e5a25b53f21e63bdef1)